test/UniswapV2FactoryTest.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import 'forge-std/Test.sol';
+
+import '../contracts/UniswapV2Factory.sol';
+import '../contracts/UniswapV2Pair.sol';
+import '../contracts/interfaces/IERC20.sol';
+import './shared/utilities.sol';
+
+contract UniswapV2FactoryTest is Test {
+    IUniswapV2Factory private factory;
+    IUniswapV2Pair private pair;
+
+    address private wallet;
+    address private other;
+
+    address[] TEST_ADDRESSES = [0x1000000000000000000000000000000000000000,
+        0x2000000000000000000000000000000000000000];
+
+    event PairCreated(address indexed token0, address indexed token1, address pair, uint256);
+
+    function setUp() public {
+        wallet = vm.addr(1);
+        other = vm.addr(2);
+        factory = new UniswapV2Factory(wallet);
+    }
+
+    function CreatePair(address token0, address token1) private {
```

Comment:
A few cosmetic things here. I would stick with the current convention of not capitalizing the first letter of a function and change this to `createPair`. Also for readability, I would place this function below the public tests that utilize it so the reader understands the context of its use prior to reading its logic.